src/EnergyPlus/DataZoneEquipment.hh

```diff
@@ -358,7 +374,7 @@ namespace DataZoneEquipment {
         int NumAvailHeatEquip;                      // Number of pieces of equipment available for heating
         int NumAvailCoolEquip;                      // Number of pieces of equipment available for cooling
         Array1D_string EquipType;
-        Array1D_int EquipType_Num;
+        Array1D<DataZoneEquipment::ZoneEquip> EquipType_Num;
```

Comment:
No `_Num`. I understand that there is already an array called `EquipType` which is an array of strings, but that array should not be necessary. You should convert things from string to `enum` as we parse the IDF and only use `enum` internally. If you want, you can put a pin in these comments and do them in a separate PR.